convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -60,16 +70,41 @@ def run(self):
         logger.task("Convert: Remove excluded packages")
         logger.info("Searching for the following excluded packages:\n")
         try:
-            pkgs_to_remove = pkghandler._get_packages_to_remove(system_info.excluded_pkgs)
-            pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove)
+            pkgs_to_remove = sorted(pkghandler._get_packages_to_remove(system_info.excluded_pkgs))
+            pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove))
 
             # TODO: Handling SystemExit here as way to speedup exception
             # handling and not refactor contents of the underlying function.
         except SystemExit as e:
             # TODO(r0x0d): Places where we raise SystemExit and need to be
             # changed to something more specific.
             #   - When we can't remove a package.
-            self.set_result(level="ERROR", id="PACKAGE_REMOVAL_FAILED", message=str(e))
+            self.set_result(level="ERROR", id="EXCLUDED_PACKAGE_REMOVAL_FAILED", message=str(e))
+            return
+
+        # shows which packages were not removed, if false, all packages were removed
+        pkgs_not_removed = sorted(frozenset(pkgs_to_remove).difference(pkgs_removed))
+        pkgs_not_removed = [str(pkg) for pkg in pkgs_not_removed]
+        pkgs_removed = [str(pkg) for pkg in pkgs_removed]
```

Comment:
This greatly diverge from https://github.com/oamg/convert2rhel/pull/888/files#diff-7bab0f1619d739dd7e18e3643a6814d444189bb25fc420206ae8e3a0b3ddde85R105, which one should we stick to?

Reply:
I think we should stick to the one in this PR

Reply:
What does `str(pkg)` look like?  The output of `get_pkg_nevra(include_zero_epoch=True)` is `0:pkg-2-3.x86_64`.

Reply:
Here, in this case, we are not using get_pkg_nevra, I believe... We are just calling str to the value we are iterating through...

Reply:
For example, this is the output: 

```
[93m(WARNING) REMOVE_EXCLUDED_PACKAGES::EXCLUDED_PACKAGES_NOT_REMOVED - The

    following packages were not removed: PackageInformation(packager='CentOS

    Buildsys <bugs@centos.org>', vendor='CentOS',

    nevra=PackageNevra(name='centos-gpg-keys', epoch='1', version='8',

    release='3.el8', arch='noarch'), fingerprint='05b555b38483c65d',

    signature='RSA/SHA256, Tue Sep 14 21:14:43 2021, Key ID 05b555b38483c65d')[0m
```

Reply:
> What does str(pkg) look like?

This is not being formatted correctly, we have to use something like: 

```python
[pkghandler.get_pkg_nevra(pkg_obj, include_zero_epoch=True) for pkg_obj in packages]
```

which is defined here: https://github.com/oamg/convert2rhel/pull/888/files#diff-7bab0f1619d739dd7e18e3643a6814d444189bb25fc420206ae8e3a0b3ddde85R27

Reply:
Ok I will add this change